server/src/main/scala/controllers/application/AuthController.scala

```diff
@@ -0,0 +1,147 @@
+package controllers.application
+
+import actors.DrtSystemInterface
+import akka.actor.ActorSystem
+import akka.event.LoggingAdapter
+import akka.http.scaladsl.model.HttpResponse
+import drt.http.ProdSendAndReceive
+import drt.shared.ErrorResponse
+import drt.users.KeyCloakClient
+import play.api.Configuration
+import play.api.libs.json.Format.GenericFormat
+import play.api.libs.json.JsResult.Exception
+import play.api.libs.json.{JsError, JsObject, Json, Writes}
+import play.api.mvc.{AbstractController, Action, AnyContent, BaseController, ControllerComponents, Headers, Result}
+import uk.gov.homeoffice.drt.auth.LoggedInUser
+import uk.gov.homeoffice.drt.auth.Roles.{ManageUsers, Role}
+import uk.gov.homeoffice.drt.ports.AirportConfig
+import upickle.default.write
+
+import scala.concurrent.{ExecutionContext, Future}
+
+
+abstract class AuthController(cc: ControllerComponents, ctrl: DrtSystemInterface) extends AbstractController(cc) {
```

Comment:
This appears to be a copy of the `WithAuth` trait. We don't want to keep both so can we remove `WithAuth`?